content/api/transmissions.apib

```diff
@@ -638,6 +638,16 @@ Only transmissions which are scheduled for future generation may be deleted.
 
 Delete all transmissions in a campaign.
 
+Please note that this endpoint immediately returns a 204 response and the
+deletion process happens in the background. When a message is deleted, you will
+receive a bounce event with a reason of "554 5.7.1 [internal] Campaign
+cancelled".
```

Comment:
I would put a comment that you can see these via Events API or through Webhooks.